docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+services:
+  arma-reforger:
+    build: .
```

Comment:
Won't this expect a Dockerfile next to compose file? I don't see a reason for this here either way.